### PR TITLE
Remove unused worflow DB name config entry

### PR DESCRIPTION
**Pull request by vincent-legoll**
This looks unused, please review...

Signed-off-by: Vincent Legoll <vincent.legoll@gmail.com>

### CODE REVIEW

File: Portal/vip-core/src/main/java/fr/insalyon/creatis/vip/core/client/view/CoreConstants.java [hunk 1]
```diff
@@ -77,7 +77,6 @@
     public static final String LAB_SIMULATION_PLATFORM_MAX = "simulation.max.platfrom";
     public static final String LAB_SIMULATION_BEGINNER_MAX = "simulation.max.beginner";
     public static final String LAB_SIMULATION_DB_HOST = "workflows.db.host";
-    public static final String LAB_SIMULATION_DB_NAME = "workflows.db.name";
     public static final String LAB_SIMULATION_DB_PORT = "workflows.db.port";
```
Comment (camarasu) [inline Portal/vip-core/src/main/java/fr/insalyon/creatis/vip/core/client/view/CoreConstants.java]: Is there a reason for keeping the host and port if the name is removed?

File: Portal/vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/Server.java [hunk 2]
```diff
@@ -182,7 +182,6 @@ private Server() {
             advancedMaxRunningSimulations = config.getInt(CoreConstants.LAB_SIMULATION_ADVANCED_MAX, Integer.MAX_VALUE);
             maxPlatformRunningSimulations=config.getInt(CoreConstants.LAB_SIMULATION_PLATFORM_MAX, Integer.MAX_VALUE);
             workflowsPath = config.getString(CoreConstants.LAB_SIMULATION_FOLDER, "/var/www/html/workflows");
-            workflowsDB = config.getString(CoreConstants.LAB_SIMULATION_DB_NAME, "/var/www/workflows.db");
             workflowsHost = config.getString(CoreConstants.LAB_SIMULATION_DB_HOST, "localhost");
             workflowsPort = config.getInt(CoreConstants.LAB_SIMULATION_DB_PORT, 1527);
```
Comment (camarasu) [inline Portal/vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/Server.java]: Same question as above

File: Portal/vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/Server.java [hunk 3]
```diff
@@ -261,7 +260,6 @@ private Server() {
             config.setProperty(CoreConstants.LAB_SIMULATION_PLATFORM_MAX, maxPlatformRunningSimulations);
            
             config.setProperty(CoreConstants.LAB_SIMULATION_FOLDER, workflowsPath);
-            config.setProperty(CoreConstants.LAB_SIMULATION_DB_NAME, workflowsDB);
             config.setProperty(CoreConstants.LAB_SIMULATION_DB_HOST, workflowsHost);
```
Comment (camarasu) [inline Portal/vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/Server.java]: Same questions

**Comment by vincent-legoll** [comment]
Looks like those:

getWorkflowsHost() & getWorkflowsPort()

are still used in:
Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/dao/h2/AbstractJobData.java

I went for the uncontroversial low hanging fruits, not enough time to investigate further cleanups...

**Comment by camarasu** [comment]
Sorry, my mistake, WorkflowsDB and WorkflowsHost have different usages, indeed. 
OK with me to merge the PR, thank you.

**Pull request merged**